Source/WebCore/page/FocusDirection.h

```diff
@@ -38,3 +40,20 @@ enum class FocusDirection: uint8_t {
 };
 
 } // namespace WebCore
+
+namespace WTF {
+
+template<> struct EnumTraits<WebCore::FocusDirection> {
```

Comment:
This would be even better in WebCoreArgumentCoders.serialization.in instead.

Reply:
Really curious how moving this away from where the enum is defined is better?

Or is there a mechanism to catch when they get out-of-sync?

Will give this a go.